pkg/controller/controller_test.go

```diff
@@ -137,6 +149,19 @@ func TestController(t *testing.T) {
 		if !test.CallCSIExpand && expandCallCount > 0 {
 			t.Fatalf("for %s: expected no csi expand call, received csi expansion request", test.Name)
 		}
+
+		if test.CallCSIExpand {
+			usedCapability := client.GetCapability()
+			if test.blockVolume {
```

Comment:
nit: `expectBlockVolume`

Also these conditionals would be easier to read if stuck together `if test.expectBlockVolum && usedCapability.GetBlock() == nil`

Reply:
fixed